docs/tutorial/building-a-theme.md

```diff
@@ -0,0 +1,1504 @@
+---
+title: Building a Theme
+---
+
+In this tutorial, you'll learn how to build a theme plugin for Gatsby. This tutorial is meant as a written companion to the [Gatsby Theme Authoring Egghead course](https://egghead.io/courses/gatsby-theme-authoring).
+
+## Overview
```

Comment:
Should we remove this heading, since it's the same level as the next one?

Reply:
Yup -- oversight, I was going to write in the overview, but the paragraph before is sufficient.